File: cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h> 

int main(void)
{
    float change;
    do
    {
        change = get_float ("Change owed: ");
    }while (change < 0.00);
     
    int cents = round (100 * change);
    int coinCount = 0;
    while(cents >= 25)
    {
        cents -= 25;
        coinCount++;
    }
    while(cents >= 10)
    {
        cents = cents - 10;
        coinCount++;
    }
    while(cents >= 5)
    {
        cents = cents - 5;
        coinCount++;
    }
    while(cents >= 1)
    {
        cents = cents - 1;
        coinCount++;
    }
    
    printf("Change %i", coinCount);
}

File: cash.py
from cs50 import get_float

while True:
    change = get_float("Change owed ")
    if change > 0.00:
        break
    
cents = round(100 * change)
coinCount  =0

while cents >=  25:
    cents -= 25
    coinCount += 1
while cents >= 10:
    cents -=10
    coinCount+=1
while cents >= 5:
    cents -= 5
    coinCount +=1
while cents >= 1:
    cents -=1
    coinCount +=1
print(f"Change {coinCount}")
    
    
    